src/file-uploader/file-uploader.interface.ts

```diff
@@ -0,0 +1,5 @@
+export class FileUploaderContent {
```

Comment:
Is this an interface or a class? If it's never instantiated it should be `export interface` if it is instantiated it should be `file-uploader-content.class.ts`

Reply:
Also maybe `FileItem` or something instead? `FileUploaderContent` isn't super descriptive of what it actually is